python_modules/libraries/dagster-k8s/dagster_k8s/external_resource.py

```diff
@@ -196,7 +206,7 @@ def build_pod_body(
         "name": pod_name,
     }
     if "labels" in meta:
-        meta["labels"] = {**get_common_labels(), **meta["labels"]}
+        meta["labels"] = {**get_common_labels(), **meta["labels"]}  # type: ignore
```

Comment:
?

Reply:
it was barking at me in vscode. maybe I have pyright misconfigured?

Reply:
![Screenshot 2023-09-12 at 4 01 57 PM](https://github.com/dagster-io/dagster/assets/28738937/401e50eb-d56b-44aa-9c0e-22551285c722)

Reply:
do whatever works, just wanted to make sure it was intentional